source/js/buyers-guide/components/primary-nav/primary-nav.scss

```diff
@@ -59,7 +59,7 @@
   height: 100vh;
   top: 0;
   left: 0;
-  width: 100vw;
+  right: 0;
```

Comment:
`left: 0;` with `right: 0;` to make an absolute positioned element go full width